src/frontend/src/pages/AdminToolsPage/BOMConfig/ManufacturerTable.tsx

```diff
@@ -16,6 +19,13 @@ const ManufacturerTable: React.FC = () => {
     error: manufacturersError
   } = useGetAllManufacturers();
   const [createModalShow, setCreateModalShow] = useState<boolean>(false);
+  const [manufacturerList, setManufacturerList] = useState<Manufacturer[]>([]);
+
+  useEffect(() => {
+    if (manufactureres) {
```

Comment:
Just realized we initially had a typo, please fix spelling to *manufacturers and *manufacturersIsLoading